packages/auth/package.json

```diff
@@ -33,7 +33,10 @@
         "react-native-msal": "*"
     },
     "dependencies": {
-        "@equinor/mad-components": "*"
+        "@equinor/mad-components": "*",
+        "expo-auth-session": "~5.0.2",
```

Comment:
Is it enough to have expo auth session as a dependency and not a peer-dependency? If users will need to install this package in their own apps, we should perhaps move it to a peer dep

Reply:
These are indeed peer dependencies. moved them to peerdependency list